src/xcode/ENA/ENAUITests/Home/ENAUITestsHome.swift

```diff
@@ -366,6 +366,18 @@ class ENAUITests_01_Home: XCTestCase {
 		snapshot("details_screen_risk_level_\(riskLevel)_risk_one_day_installation_\(installationDays)days_\(String(format: "%04d", (screenshotCounter.inc() )))")
 	}
 	
+	func test_screenshot_homescreen_pcr_rat_negative() throws {
+		app.setPreferredContentSizeCategory(accessibility: .accessibility, size: .XS)
+		app.launchArguments.append(contentsOf: ["-showTestResultCards", TestResult.negative.stringValue])
```

Comment:
Shouldn't this be something like: 
```suggestion
		app.launchArguments.append(contentsOf: ["-showTestResultCards", "YES"])
```
?